rocAL/rocAL/include/api/rocal_api_types.h

```diff
@@ -55,21 +51,6 @@ struct TimingInfo
     long long unsigned process_time;
     long long unsigned transfer_time;
 };
```

Comment:
let's retain this change. We will need in future.

Reply:
@fiona-gladwin keypoints changes needs to be reintroduced

Reply:
Done